syntax = "proto3";

package Echo;

option cc_generic_services = true;

message EchoRequest{
    bytes data = 1;
}

message EchoResponse{
    bytes data = 1;
}

service EchoService{
    rpc DoEcho(EchoRequest) returns(EchoResponse);
}